packages/rum-core/src/performance-monitoring/transaction-service.js

```diff
@@ -180,7 +176,7 @@ class TransactionService {
     tr.onEnd = () => this.handleTransactionEnd(tr)
 
     if (__DEV__) {
-      this._logger.debug('TransactionService.startTransaction', tr)
+      this._logger.debug('startTransaction', { name: tr.name, type: tr.type })
```

Comment:
Also having transaction helps here, you can include name and type in the message `startTransaction(${name},${type})`

Reply:
Changed the format now.